packages/terra-avatar/src/Avatar.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import TerraImage from 'terra-image';
+import styles from './Avatar.scss';
+
+const cx = classNames.bind(styles);
+
+const AvatarVariants = {
+  FACILITY: 'facility',
+  USER: 'user',
+};
+
+const propTypes = {
+  /**
+   * The text content that specifies the alternative text for the image.
+   */
+  alt: PropTypes.string,
+  /**
+   * The text content of the aria-label for accessibility.
+   */
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
On the original tech design, this wasn't a prop. Is there a way this component can populate the aria-label for you instead?

Reply:
We had originally considered doing this, but having the component populate the label didn't provide enough information to make the label useful from an accessibility standpoint. Using a required prop, the consumer could send in the user's/facility's name which provides much more value than "User Avatar" or "JS". I would say as is, this is what we believed to be the best approach. If you have a suggestion about how the component could provide a valuable label, particularly considering the complications that arise based on whether this is an _image_, _initials_, or _icon_ `Avatar` then i’d happily make the change.

Reply:
That makes more sense. Perhaps for our documentation and test examples, can you change some of them to use names such as John Doe or a medical center?

Reply:
Also, let's make this not required.

Reply:
Updated in this [commit](https://github.com/cerner/terra-core/commit/901237e7a6938c3655ffa65a772ace2bb6d7144e).